systems/registry/org/pkg/server/org.go

```diff
@@ -563,3 +708,30 @@ func pbRoleTypeToDb(role pb.RoleType) db.RoleType {
 		return db.Member
 	}
 }
+
+func pbInvitationStatusToDb(status pb.InvitationStatus) db.InvitationStatus {
+	switch status {
+	case pb.InvitationStatus_PENDING:
+		return db.Pending
+	case pb.InvitationStatus_ACCEPTED:
+		return db.Accepted
+	case pb.InvitationStatus_REJECTED:
+		return db.Rejected
+	default:
+		return db.Pending
+	}
+
+}
+
+func generateInvitationLink() (string, error) {
+	expirationTime := time.Now().Add(3 * 24 * time.Hour) // Set the expiration time to three days from now
```

Comment:
We are generating expirationTime twice. This might causes mismatch between the 2 expiration times and might cause some bugs.

Reply:
I have fixed this by passing expiration time as a param to the function